### DISCUSSION TITLE
vibration/rumble didn't work with bluetooth but work with usb

### DISCUSSION THREAD

**Discussion by inibintang** [nefarius/DsHidMini#265] [Support]
BthPS3 Bluetooth Drivers v2.6.174
DsHidMini Driver v2.2.282

ps3 sixaxis CECHZC2E B1

HID : xinput

**Comment by s1davide** [comment]
I have the same issue with ps3 sixaxis CECHZC2U A1

**Comment by nefarius** [comment]
[It's the controllers fault, can't do anything in software about it.](https://vigem.org/projects/BthPS3/About-Controller-Compatibility/)

**Reply by s1davide** [reply]
I really appreciate your answer, but I have certain questions, I had read in a previous issue that you had solved the vibration in Bluetooth connection, the drivers were updated and deactivated this feature? Is there any way to fix it or alternative way of use? Excuse my ignorance about this topic, and thank you for your collaboration in these projects

**Reply by nefarius** [reply]
In summary: some 3rd party controllers for some unknown reason do not respond the same way to the packets they get sent like the original Sony hardware does. Why? No idea. Is it fixable? No idea, I don't have the time or interest to dump hours upon hours into figuring out why garbage aftermarket hardware ain't playing nice with my hobby projects. That's it.

**Reply by nefarius** [reply]
> I had read in a previous issue that you had solved the vibration in Bluetooth connection, the drivers were updated and deactivated this feature?

No idea what you mean by that, do you have a link?

**Reply by s1davide** [reply]
I was referring to this issue https://github.com/ViGEm/DsHidMini/issues/4 , I think the user had a control with the same model, only mine has an additional "A1", that must be why it worked for him and then it didn't for me?

**Reply by nefarius** [reply]
That issue got closed as resolved. Again, please elaborate on your claim that something got "deactivated" in the past as I do not see it anywhere.

**Reply by s1davide** [reply]
It is not a statement, it was just a supposition as I am telling you, to know if you could help me in any way, I do not have your level of software development, and I did not know the reason why someone with such a similar model could solve the vibration problem.

**Reply by s1davide** [reply]
The assumption is that sometimes people disable features because they give a lot of trouble or are unstable, I was just wondering if I am doing something wrong with such a similar model

**Reply by nefarius** [reply]
Nothing in DsHidMini or BthPS3 is disabled on purpose. [Also see this.](https://vigem.org/projects/BthPS3/About-Controller-Compatibility/). That's all there is to it. I have nothing more to add.

**Reply by s1davide** [reply]
Ok, fine, thanks, I was just asking for help, don't take it as if I were complaining or criticizing.

**Reply by nefarius** [reply]
I don't, I'm simply stating the facts and linking to docs I wrote exactly for these kind of questions.
